include/networkit/centrality/ApproxElectricalCloseness.hpp

```diff
@@ -109,15 +138,11 @@ class ApproxElectricalCloseness final : public Centrality {
     std::vector<index> topOrder;
 
     // Non-normalized approx effective resistance, one per thread.
-    // Values could be negative, so we use signed integers.
-    std::vector<std::vector<int64_t>> approxEffResistanceGlobal;
```

Comment:
Likely the documentation needs to be adapted. These values seems now normalized due to the change from `int64_t` to `double`.